packages/blade/src/components/Alert/Alert.tsx

```diff
@@ -235,15 +230,26 @@ const Alert = ({
     secondaryActionParams.rel = actions.secondary.rel;
   }
   const secondaryAction = actions?.secondary ? (
-    <SecondaryActionWrapper>
+    <Box marginRight="spacing.4" display={isReactNative ? 'flex' : 'inline-flex'}>
       <BaseLink size={textSize} contrast={contrast} intent={intent} {...secondaryActionParams}>
         {actions.secondary.text}
       </BaseLink>
-    </SecondaryActionWrapper>
+    </Box>
   ) : null;
 
-  const _actions =
-    primaryAction || secondaryAction ? (
+  // For certain cases we wish to render actions inline with text content
+  const showActionsHorizontal = isFullWidth && isDesktop;
+
+  const actionsHorizontal =
+    showActionsHorizontal && (primaryAction || secondaryAction) ? (
```

Comment:
```suggestion
    showActionsHorizontal && (primaryAction || secondaryAction || isDismissable) ? (
```

shall check for dismissable as well?

Reply:
Dismissible check isn't added because we don't want to render the container of actions at all if both primary and secondary actions aren't present. Say we add this in the OR conditional and none of primary and secondary action are added, it would still render an empty `Box` (container for actions)